consent/strategy_default.go

```diff
@@ -109,7 +109,8 @@ func (s *DefaultStrategy) matchesValueFromSession(ctx context.Context, c fosite.
 
```

Comment:
These changes here are cleaning up variables and so on right? I think they should be made as a separate PR because it makes it harder for me to review what belongs to what. Thank you!